apps/src/p5lab/P5Lab.js

```diff
@@ -1078,7 +1079,12 @@ P5Lab.prototype.initInterpreter = function(attachDebugger = true) {
     }
 
     if (this.isSpritelab) {
-      this.coreLibrary = new CoreLibrary(this.p5Wrapper.p5);
+      if (this.level.blockPools.includes('PoemBot')) {
+        this.coreLibrary = new PoemBotLibrary(this.p5Wrapper.p5);
+      } else {
+        this.coreLibrary = new CoreLibrary(this.p5Wrapper.p5);
+      }
```

Comment:
2 thoughts here:

[similar to [this comment](https://github.com/code-dot-org/code-dot-org/pull/41673#discussion_r677843212) on another PR] i think we should export all libraries from a `./spritelab/libraries/index.js` file and organize libraries within that directory. (this can/should be follow-up work.)

i think we should offload this logic to a "library factory." right now, it could just be a function within P5Lab that creates `this.coreLibrary`, but i could see it becoming large/complicated enough that we create a separate factory at some point. [this article](https://medium.com/@thebabscraig/javascript-design-patterns-part-1-the-factory-pattern-5f135e881192) is a pretty good showcase of a simple factory like what we'd want/need here